src/index.js

```diff
@@ -29,6 +30,7 @@ if (isProduction) {
   store = createStore(reducer, initialState, devTools);
 }
 store.dispatch(setContext('iconContext', iconContext));
+store.dispatch(setContext('playlistContext', playlistContext));
```

Comment:
index-static.js bør oppdateres tilsvarende, inkludert å få sendt in store via <Provider>. (Det er vel litt mangelfullt fra tidligere PR også). Felles funksjonalitet kan splittes ut i egen fil (f.eks. index-common.js?)

Tilsvarende bør router-static.js også oppdateres.